applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -539,10 +539,10 @@ static void discover_sink_cb(struct bt_conn *conn, struct bt_codec *codec, struc
 
 #if CONFIG_STREAM_BIDIRECTIONAL
 static void discover_source_cb(struct bt_conn *conn, struct bt_codec *codec, struct bt_audio_ep *ep,
-			       struct bt_audio_sink_discover_params *params)
+			       struct bt_audio_discover_params *params)
 {
 	int ret = 0;
-	uint8_t ep_index = 0;
+
```

Comment:
No need for a newline here, but no need to change it if it interferes with the CI process. Important to get this in ASAP